artifact/v3/version/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+
+# Test the parser
+add_executable(artifact_version_parser_test EXCLUDE_FROM_ALL
+  version.cpp
+  version_test.cpp
+)
+target_link_libraries(artifact_version_parser_test PRIVATE
+  GTest::gtest_main
+  gmock
+  common_io
+  common_error
+  common_json
```

Comment:
Yeh, it's good you pointed this out, I forgot to mention it.

So I _did not_ make a module out of this, since it is so small, in which case, the test needs this to link.

However maybe this was a bad idea. I really don't know. 

Since it caught you by surprise, maybe I should make a separate library out of it, it just felt a bit extensive.

I'll let you decide :pray: